youtube_dl/extractor/twitch.py

```diff
@@ -26,18 +27,18 @@
     try_get,
     unified_timestamp,
     update_url_query,
-    urlencode_postdata,
     url_or_none,
     urljoin,
 )
 
 
 class TwitchBaseIE(InfoExtractor):
-    _VALID_URL_BASE = r'https?://(?:(?:www|go|m)\.)?twitch\.tv'
+    _VALID_URL_BASE = r'https?://(?:(?:www|go|m|passport)\.)?twitch\.tv'
```

Comment:
No.

Reply:
@dstftw could you clarify, please? `passport` is the new subdomain. But, I just tested and the updates work fine without it. Would you like me to remove `passport`?

Reply:
Have you read where `_VALID_URL_BASE` is used in code at all?